crates/stc_ts_type_checker/tests/conformance/expressions/functionCalls/callWithSpread2.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 5,
-    matched_error: 7,
-    extra_error: 3,
+    required_error: 6,
+    matched_error: 6,
+    extra_error: 4,
```

Comment:
Can you fix this regression, too?

Reply:
Fixed! Unfortunately, I had to revert a piece of code that enabled `iteratorSpreadInCall10.ts` to pass. But I think that's a separate issue anyway.